package.json

```diff
@@ -4,8 +4,9 @@
     "dev": "next dev",
     "build": "next build",
     "start": "next start",
-    "test": "jest",
-    "test:watch": "jest --watchAll",
+    "test": "jest --passWithNoTests",
```

Comment:
What was this param added to the script?

Reply:
When pulling from the develop's most recent changes (removing Home component and test folder), I was geting the following error in my PR when trying to push into the storybook-docs branch:

```Run pnpm test

> @ test /home/runner/work/gridiron-survivor/gridiron-survivor
> jest

No tests found, exiting with code 1
Run with `--passWithNoTests` to exit with code 0
In /home/runner/work/gridiron-survivor/gridiron-survivor
  29 files checked.
  testMatch: **/__tests__/**/*.[jt]s?(x), **/?(*.)+(spec|test).[tj]s?(x) - 0 matches
  testPathIgnorePatterns: /node_modules/, /.next/ - 29 matches
  testRegex:  - 0 matches
Pattern:  - 0 matches
 ELIFECYCLE  Test failed. See above for more details.
Error: Process completed with exit code 1.```

So after consulting with Mai to debug the error, after some research in regards to this particular error, she failed to duplicate the same error as my local machine/PR.  Our inference is that my machine still remembers the tests, and therefore is still looking for them; therefore it fails the Docker test because there are no tests to look for.

Reply:
I believe this is ok for now until we add our 1st test. Then, we need to remove this.